### PR TITLE
Figma documentation links are outdated

### PR DESCRIPTION
**Pull request by jdde**
Hi guys,

I just noticed, that all figma links in this file lead to a 404 error page.
They're probably outdated.

https://github.com/primer/design/blob/main/content/tools/figma.mdx

Tried to find the correct links & corrected documentation

### CODE REVIEW

File: content/tools/figma.mdx [hunk 1]
```diff
@@ -19,12 +19,11 @@ Design teams at GitHub use Figma as the single source of truth, collaboration, a
 
 ## Team libraries
 
-GitHub uses Figma to distribute Primer through team libraries. Each library covers a different part of the system, allowing a designer to only use what's needed. Any team within GitHub's Figma organization can create and distribute their own team/project specific libraries, but all of the core asset libraries for Primer can be found in the Primer team's "[Core libraries](https://www.figma.com/files/797266083973274396/project/8172024/Core-libraries)" project. They include the following:
+GitHub uses Figma to distribute Primer through team libraries. Each library covers a different part of the system, allowing a designer to only use what's needed. Any team within GitHub's Figma organization can create and distribute their own team/project specific libraries, but all of the core asset libraries for Primer can be found in the Primer team's "[Core libraries](https://www.figma.com/@primer)" project. They include the following:
```
Comment (lukasoppermann) [inline content/tools/figma.mdx:22]:
```suggestion
GitHub uses Figma to distribute Primer through team libraries. Each library covers a different part of the system, allowing a designer to only use what's needed. Any team within GitHub's Figma organization can create and distribute their own team/project specific libraries, but all of the core asset libraries for Primer can be found in the Primer team's "[Core libraries (internal)](https://www.figma.com/files/797266083973274396/project/8172024/Core-libraries)" project or the [primer community page](https://www.figma.com/@primer). They include the following:
```

File: content/tools/figma.mdx [hunk 2]
```diff
@@ -19,12 +19,11 @@ Design teams at GitHub use Figma as the single source of truth, collaboration, a
...
 
-- [Primer Primitives](https://www.figma.com/file/B5XPE8IwGPIZDAvN7jqWqx/Primer-Primitives): All of the color, type, and spacing styles used within Primer.
-- [Primer Web](https://www.figma.com/file/GCvY3Qv8czRgZgvl1dG6lp/Primer-Web): UI components used to design for GitHub's web interfaces.
-- [Primer Mobile](https://www.figma.com/file/csCViryWkkXwBsK1aLrO0X/Primer-Mobile): UI components used to design for GitHub's native mobile platforms.
-- [Octicons](https://www.figma.com/file/1ljgTFkT5NKNRfq5hw07JQ/Octicons): GitHub's custom icon library.
+- [Primer Primitives](https://www.figma.com/community/file/854766928300977832): All of the color, type, and spacing styles used within Primer.
```
Comment (lukasoppermann) [inline content/tools/figma.mdx:24]:
```suggestion
- Primer Primitives ([Internal](https://www.figma.com/file/B5XPE8IwGPIZDAvN7jqWqx/Primer-Primitives) | [Community](https://www.figma.com/community/file/854766928300977832)): All of the color, type, and spacing styles used within Primer.
```

You could change it to have an Internal and a community link for the items. Mobile is internal only at the moment.

**Comment by lukasoppermann** [comment]
Hey @jdde thank you for the contribution. 🙏 
The links are actually not outdated but internal.

This is a bit of the dance we have because everything in Figma exists in the open and internally.  
I added a few suggestions on how you could update your PR so that I can merge it. Let me know if you have any questions.

**Review by camertron — APPROVED** [review]

File: content/tools/figma.mdx [hunk 3]
```diff
@@ -19,12 +19,11 @@ Design teams at GitHub use Figma as the single source of truth, collaboration, a
...
 
-- [Primer Primitives](https://www.figma.com/file/B5XPE8IwGPIZDAvN7jqWqx/Primer-Primitives): All of the color, type, and spacing styles used within Primer.
-- [Primer Web](https://www.figma.com/file/GCvY3Qv8czRgZgvl1dG6lp/Primer-Web): UI components used to design for GitHub's web interfaces.
-- [Primer Mobile](https://www.figma.com/file/csCViryWkkXwBsK1aLrO0X/Primer-Mobile): UI components used to design for GitHub's native mobile platforms.
-- [Octicons](https://www.figma.com/file/1ljgTFkT5NKNRfq5hw07JQ/Octicons): GitHub's custom icon library.
+- Primer Primitives ([Internal](https://www.figma.com/file/B5XPE8IwGPIZDAvN7jqWqx/Primer-Primitives) | [Community](https://www.figma.com/community/file/854766928300977832)): All of the color, type, and spacing styles used within Primer.
+- [Primer Web](https://www.figma.com/community/file/854767373644076713): UI components used to design for GitHub's web interfaces.
```
Comment (lukasoppermann) [inline content/tools/figma.mdx:25]:
```suggestion
- [Primer Mobile (internal only)](https://www.figma.com/file/csCViryWkkXwBsK1aLrO0X/Primer-Mobile): UI components used to design for GitHub's native mobile platforms.
- Primer Web ([(internal)](https://www.figma.com/community/file/854767373644076713) | [Community](https://www.figma.com/community/file/854767373644076713)): UI components used to design for GitHub's web interfaces.
```

**Pull request merged**